api/routes/index.js

```diff
@@ -17,12 +21,34 @@ route.get('/', limiter.rate100, (req, res) => {
   });
 });
 
-// hello world!
+// Routes
+
+// hello world
 route.get('/hello', (req, res) => res.send('Hello World!'));
 
-// HEALTHCHECK -- is this a requirement?
-// Healthcheck Routes
-route.get('/health', limiter.rate250, healthcheck.db);
+// healthcheck
+route.get('/health', limiter.rate250, healthcheck.healthcheck);
+
+// all albums
+route.get('/albums', limiter.rate250, album.allAlbums);
+
+// by album id
+route.get('/albums/:albumID', limiter.rate250, album.byAlbumID);
+
+// all tracks in an album by id
+route.get('/tracks?albumid={albumid}', limiter.rate100, track.multipleTracks);
```

Comment:
you might not need this. You will still get the query params in the route you have on line 42.

Reply:
you're right- it doesn't. I've removed it but will add the path to the documentation